content/en/docs/concepts/data-collection.md

```diff
@@ -41,11 +45,180 @@ The Collector is made up of the following components:
+```yaml
+processors:
+  batch:
+    timeout: 10s
+    send_batch_size: 10000
+    send_batch_max_size: 11000
+```
+
+**Filter processor**
+
+The filter processor makes it possible to include or exclude telemetry data using the configuration parameters
+provided. It can be configured to match with `strict` or `regexp` names. The filter can be further scoped by specifying
+`resource_attributes`.
+
+**Memory limiter processor**
+
+The memory limiter processor enables the user to control the amount of memory the collector consumes. This ensures that the 
+collector is conscious of resource consumption, and does everything it can to avoid running out of memory. The memory
+limiter has to be the first processor to configure in the pipeline.
+
+The configuration below shows how to set the memory limiter to use up to 300 Mib via the `limit_mib` parameter with a
+difference of 50 Mib between soft and hard limits configured via the `spike_limit_mib`:
+```yaml
+processors:
+  memory_limiter:
+    check_interval: 5s
+    limit_mib: 300
+    spike_limit_mib: 50
+  extensions:
+    memory_ballast:
+      size_mib: 150
+```
+
+**Probabilistic sampling processor**
+
+The probabilistic sampling processor is used to reduce the number of traces exported from the collector. This is 
+done by specifying a sampling percentage which determines the threshold percentage that should be preserved. An example
+of a probabilistic sampler is shown below:
+```yaml
+processors:
+  probabilistic_sampler:
+    sampling_percentage: 50
+    hash_seed: 123456
+```
+
+The `hash_seed` parameter determines how the collector should hash the trace IDs for determining which traces to process.
+
+**Resource processor**
+
+The resource processor enables users to modify attributes like the attribute processor but instead of updating attributes
+on individual spans, metrics, or logs, it updates attributes of the resource associated with the telemetry data.
+
+**Span processor**
+
+The job of the span processor is to manipulate the names or attributes of spans. It can extract the attributes of a span 
+and update its name based on those attributes. It supports   include` and `exclude` configuration parameters for filtering
+spans. Some span processors are used to change the collector's behavior.
+
+Learn how to configure processors in the
+[configuration documentation](/docs/collector/configuration/#processors).
+
+### Exporters
+The job of the exporter is to receive data in the internal collector format, convert it into the output format, and 
+transport it to one or more specified destinations. Multiple exporters of the same type can be configured to transport
+data to different destinations as required. Similarly, multiple exporters can be configured in the same pipeline to 
+transport data to multiple destinations.
+
+The table below shows the available exporters and the signals they support:
```

Comment:
Similar to other comments, I think it'd probably best to not include this table here for now.